src/deferred.js

```diff
@@ -305,64 +322,43 @@ jQuery.extend( {
 	},
 
 	// Deferred helper
-	when: function() {
-		var method, resolveContexts,
-			i = 0,
-			resolveValues = slice.call( arguments ),
-			length = resolveValues.length,
+	when: function( singleValue ) {
+		var // count of uncompleted subordinates
```

Comment:
The comment position seems a little off-kilter here. Maybe add a newline or tab? Most likely a newline.